wandb/sdk_py27/internal/sender.py

```diff
@@ -747,18 +771,22 @@ def send_output(self, data):
             self._fs.push(filenames.OUTPUT_FNAME, line)
             self._partial_output[stream] = ""
 
-    def _update_config(self):
+    def _update_config(self, data, debounce=False):
```

Comment:
why the special case?  lets debounce config all the time.

Reply:
data isnt used.  dont pass it around.

Reply:
wanted to be conservative with the initial rollout of the debouncer incase there were any load problems. But i removed this for now and have everything debouncing.